src/h2load.cc

```diff
@@ -1543,17 +1686,16 @@ int main(int argc, char **argv) {
   if (!config.is_rate_mode() && config.nclients == 1) {
     config.nthreads = 1;
   }
-  ssize_t seconds = 0;
 
   if (config.is_rate_mode()) {
 
     // set various config values
     if (config.nreqs < config.nconns) {
-      seconds = c_time;
+      config.seconds = c_time;
```

Comment:
I'm curious why seconds is now data field in config object.  What is the purpose for this change?